### PR TITLE
[ADD] Provide code for chapter 11

### PR DESCRIPTION
**Pull request by PhilippKuhlmay**

### CODE REVIEW

File: Ch11/Step_4/site_promotiontour/Configuration/TypoScript/constants.typoscript [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  layoutRootPath = EXT:site_promotiontour/Resources/Private/Layouts/Page/
+  partialRootPath = EXT:site_promotiontour/Resources/Private/Partials/Page/
+  templateRootPath = EXT:site_promotiontour/Resources/Private/Templates/Page/
+}
+# content
+styles.templates {
+  layoutRootPath = EXT:site_promotiontour/Resources/Private/Layouts/ContentElements/
+  partialRootPath = EXT:site_promotiontour/Resources/Private/Partials/ContentElements/
+  templateRootPath = EXT:site_promotiontour/Resources/Private/Templates/ContentElements/
+}
```
Comment (tmfrstn) [inline Ch11/Step_4/site_promotiontour/Configuration/TypoScript/constants.typoscript:14]: Indent size needs change

File: Ch11/all_code_from_chapter-11.txt [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+## Step 4: Adding a language to your template
+
+// File: site_promotiontour/Configuration/TypoScript/setup.typoscript
+
+page = PAGE
+page {
+    ...
+    10 = FLUIDTEMPLATE
+    10.templateRootPaths.10 = {$page.templates.templateRootPath}
+    10.partialsRootPaths.10 = {$page.templates.partialRootPath}
```
Comment (tmfrstn) [inline Ch11/all_code_from_chapter-11.txt:13]: 10.partial**s**RootPaths

File: Ch11/Step_4/site_promotiontour/Configuration/TypoScript/constants.typoscript (hunk 1 above)
Comment (PhilippKuhlmay) [reply Ch11/Step_4/site_promotiontour/Configuration/TypoScript/constants.typoscript]: Fixed it

File: Ch11/all_code_from_chapter-11.txt (hunk 2 above)
Comment (PhilippKuhlmay) [reply Ch11/all_code_from_chapter-11.txt]: Fixed it

**Review by tmfrstn — APPROVED** [review]

**Pull request merged**